### PR TITLE
add profiler

### LINKED ISSUE #11: display recorded events in symfony profiler
https://symfony.com/doc/current/profiler/data_collector.html

### PR DESCRIPTION
**Pull request by DavidBadura**
fix #11 

Bildschirmfoto 2022-01-02 um 17 07 54

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "phpunit/phpunit": "^9.5.10",
     "roave/security-advisories": "dev-master",
     "symfony/messenger": "^4.4.34|5.4.0|^6.0.0",
+    "symfony/web-profiler-bundle": "^4.4.34|5.4.0|^6.0.0",
```
Comment (DanielBadura) [inline composer.json]:
```suggestion
    "symfony/web-profiler-bundle": "^4.4.34|^5.4.0|^6.0.0",
```

File: src/DataCollector/EventCollector.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+namespace Patchlevel\EventSourcingBundle\DataCollector;
+
+use Patchlevel\EventSourcing\Aggregate\AggregateChanged;
+use Symfony\Bundle\FrameworkBundle\DataCollector\AbstractDataCollector;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Throwable;
+
+/**
+ * @psalm-type DataType = array{
```
Comment (DanielBadura) [inline src/DataCollector/EventCollector.php:20]: maybe name it EventList

File: src/Resources/views/Collector/template.html.twig [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+    {{ include('@WebProfiler/Profiler/toolbar_item.html.twig') }}
+{% endblock %}
+
+{% block menu %}
+    <span class="label">
+        <span class="icon">
+            {{ include('@PatchlevelEventSourcing/Collector/icon.svg') }}
+        </span>
+        <strong>Event-Sourcing</strong>
```
Comment (DanielBadura) [inline src/Resources/views/Collector/template.html.twig]:
```suggestion
        <strong>Event Sourcing</strong>
```

File: src/Resources/views/Collector/template.html.twig [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+            <tr>
+                <th colspan="2">
+                    {{ _.display_class(event.class) }}
+                </th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <td class="text-bold">Aggregate ID</td>
+                <td>{{ event.aggregateId }}</td>
```
Comment (DanielBadura) [inline src/Resources/views/Collector/template.html.twig:80]: We cannot add the information here for which `aggregate` this was triggered, can we?

**Review by DanielBadura — APPROVED** [review]

File: src/Resources/views/Collector/template.html.twig (hunk 4 above)
Comment (DavidBadura) [reply src/Resources/views/Collector/template.html.twig:80]: Sadly we don't have the information in the events.

**Pull request merged**